test_integration/01_chain_setup/01_chain_setup.js

```diff
@@ -0,0 +1,113 @@
+// Copyright 2019 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ----------------------------------------------------------------------------
+//
+// http://www.simpletoken.org/
+//
+// ----------------------------------------------------------------------------
+
+'use strict';
+
+const { assert } = require('chai');
+const Web3 = require('web3');
+
+const shared = require('../shared');
+const { ChainSetup } = require('../../index');
+
+const MockContractsDeployer = require('../../tests/utils/MockContractsDeployer');
+
+describe('ChainSetup', () => {
+  let caMockToken;
```

Comment:
What is `ca`? 🤔

Reply:
`ca` is contract address. I've usually cleaned it up to `address`, but looks like I missed this one.